Release automation tries to use non-existent release-* branch on releases repo

When doing a non-RC release from `devel` (any `make release` step) the release automation fails with:

```
[releases]$ gh pr create --repo submariner-io/releases --head submariner-io:releasing-0.10.1 --base release-0.10 --title Advancing 0.10.1 release to status: admiral --body Advancing 0.10.1 release to status: admiral
pull request create failed: GraphQL error: Head sha can't be blank, Base sha can't be blank, No commits between release-0.10 and releasing-0.10.1, Base ref must be a branch
make: *** [Makefile:49: release] Error 1
[releases]$ make --debug=b release
```

Replacing the `base` target with `devel` makes things work.

--- a/Makefile
+++ b/Makefile
@@ -2,6 +2,7 @@ BASE_BRANCH ?= devel
 GIT_EMAIL ?= release@submariner.io
 GIT_NAME ?= Automated Release
 SHELLCHECK_ARGS := scripts/lib/* scripts/*.sh
+export BASE_BRANCH
 export GIT_EMAIL
 export GIT_NAME
 

--- a/scripts/release.sh
+++ b/scripts/release.sh
@@ -49,7 +49,8 @@ function set_status() {
 function create_pr() {
     local branch="$1"
     local msg="$2"
-    local base_branch="${release['branch']:-devel}"
+    # shellcheck disable=SC2153
+    local base_branch="${BASE_BRANCH}"
     local pr_to_review
     local project
 

